testdata/project-v3/PROJECT

```diff
@@ -3,19 +3,45 @@ layout: go.kubebuilder.io/v3-alpha
 projectName: project-v3
 repo: sigs.k8s.io/kubebuilder/testdata/project-v3
 resources:
-- crdVersion: v1
+- api:
+    namespaced: true
+    version: v1
+  controller: true
+  domain: testproject.org
   group: crew
   kind: Captain
   version: v1
-  webhookVersion: v1
-- crdVersion: v1
+  webhooks:
+    defaulting: true
+    validating: true
+    version: v1
+- api:
+    namespaced: true
+    version: v1
+  controller: true
+  domain: testproject.org
   group: crew
   kind: FirstMate
   version: v1
-  webhookVersion: v1
-- crdVersion: v1
+  webhooks:
+    conversion: true
+    version: v1
+- api:
+    version: v1
+  controller: true
+  domain: testproject.org
   group: crew
   kind: Admiral
   version: v1
-  webhookVersion: v1
+  webhooks:
+    defaulting: true
+    version: v1
+- api:
+    namespaced: true
+    version: v1
+  controller: true
+  domain: testproject.org
+  group: crew
+  kind: Laker
+  version: v1
```

Comment:
```bash
$kb create api --group crew --version v1 --kind Laker --controller=true --resource=false --make=false
```

Reply:
I think we are still unable to identify `--resource=false`, maybe instead of a `resources[].api.external` boolean we should have a `resource[].api.generated` that is set to true if `--resource` flag was set.

Reply:
Opted to remove the whole `api` field for `--resource=false` and remove the `external` field as it is no longer needed.